test/parallel.jl

```diff
@@ -202,6 +202,9 @@ if haskey(ENV, "PTEST_FULL")
     @test ups == bytestring(UInt8[uint8(c) for c in pmap(x->uppercase(x), s)])
     @test ups == bytestring(UInt8[uint8(c) for c in pmap(x->uppercase(char(x)), s.data)])
 
+    # specify pids
+    @test sort(Base.workers()[1:2]) == unique(pmap(x->myid(), 1:10, pids = Base.workers()[1:2]))
+
```

Comment:
This test can be moved out of the enclosing `if`, i.e., can be executed in all runs since it is quite travis/appveyor friendly.